amplify/backend/function/updateContributorsWithReputation/src/utils.js

```diff
@@ -281,6 +282,29 @@ const updateReputationInDomain = async ({
   );
 };
 
+const calculatePercentageReputation = (userReputation, totalReputation) => {
+  if (!userReputation || !totalReputation) return null;
+  const userReputationNumber = BigNumber.from(userReputation);
+  const totalReputationNumber = BigNumber.from(totalReputation);
+
+  const reputationSafeguard = BigNumber.from(100).pow(2);
+
+  if (
+    userReputationNumber.isZero() ||
+    totalReputationNumber.isZero() ||
+    userReputationNumber.mul(reputationSafeguard).lt(totalReputationNumber)
+  ) {
+    return 0;
+  }
+
+  const reputation = userReputationNumber
+    .mul(reputationSafeguard)
+    .div(totalReputationNumber)
+    .toNumber();
+
+  return reputation / 10 ** 2;
```

Comment:
Is there a reason we don't divide back to `reputationSafeguard` value before the `.toNumber()` conversion?